public/views/backup.html

```diff
@@ -30,11 +30,11 @@
     <div class="row" ng-show="(index.n>1 && index.m != index.n )">
       <div class="m10t columns size-14 text-gray">
         <span translate>
-          To restore this {{index.m}}-{{index.n}} shared wallet you will need
+          To restore this {{index.m}}-{{index.n}} <b>shared</b> wallet you will need
```

Comment:
In this line is the same phrase :

https://github.com/gabrielbazan7/copay/commit/b128be58c17e1e4f64b75e1ec20891762a80f4a1#diff-8d05805f74f9ce3f1a53ae0c3ffd4acbL46

Also, I check that the translation exist in crowdin.